java.source.base/src/org/netbeans/modules/java/source/save/CasualDiff.java

```diff
@@ -200,13 +210,13 @@
      * on how guarded blocks (or other divisor of the source) is mapped into the changed text and not generate diffs across
      * such a boundary.
      */
-    private Map<Integer, Integer>   blockSequenceMap = new LinkedHashMap<Integer, Integer>();
+    private Map<Integer, Integer> blockSequenceMap = new LinkedHashMap<>();
 
     private Iterator<Integer>    boundaries;
     private int nextBlockBoundary = -1;
 
     protected CasualDiff(Context context, DiffContext diffContext, TreeUtilities treeUtilities, Map<Tree, ?> tree2Tag, Map<Tree, DocCommentTree> tree2Doc, Map<?, int[]> tag2Span, Set<Tree> oldTrees) {
-        diffs = new LinkedHashSet<Diff>();
+        diffs = new LinkedHashSet<>();
```

Comment:
Removing <DIff> here makes it harder to understand that diff is a linked hash set of Diff. So please keep <Diff>  here.